### PR TITLE
new package: afew-1.3.0 + dependencies

### PR DESCRIPTION
**Pull request by crater2150**
I'm not sure how to handle manpages for python-dkimpy, which is built for python2 and python3. In the current version of the template, only the python2 subpackage contains the manpages, which is consistent with how youtube-dl handles it. All other python cross-version packages I checked had no man pages at all.

### CODE REVIEW

File: srcpkgs/afew/template [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+hostmakedepends="git python3-setuptools python3-Sphinx pkg-config"
+depends="python3-setuptools notmuch-python3 python3-dkimpy"
+short_desc="An initial tagging script for notmuch mail"
+maintainer="Alexander Gehrke <void@qwertyuiop.de>"
+license="ISC"
+homepage="https://github.com/afewmail/afew"
+_gitsrc="${homepage}.git"
+
+do_fetch() {
+	git clone ${_gitsrc} ${wrksrc} --branch ${version}
```
Comment (Johnnynator) [inline srcpkgs/afew/template]: No. Use the release tarball instead of git.

Comment (crater2150) [reply srcpkgs/afew/template]: Using the release tarball causes a build failure with a message saying that github's release tarballs do not contain some required metadata and a clone should be used.

Comment (crater2150) [reply srcpkgs/afew/template]: ok, I checked Gentoo's ebuild, the Pipy mirror seems to contain all required data for the build, updated it.

File: srcpkgs/python3-dkimpy [hunk 2]
```diff
@@ -0,0 +1 @@
+python-dkimpy/
```
Comment (CameronNemo) [inline srcpkgs/python3-dkimpy]: Remove the / at the end here

**Comment by crater2150** [comment]
Anyone have an idea, why the python-based documentation tool would fail on aarch64-musl only? I don't have much experience with python or crossbuilding, but `EOFError` sounds to me like some files got corrupted?

**Comment by lemmi** [comment]
All green now, travis just needed a second attempt :)

**Pull request merged**